src/components/TextareaField/TextareaField.tsx

```diff
@@ -114,9 +178,13 @@ export const TextareaField: TextareaFieldType = forwardRef(
       <div className={componentClassName}>
         {shouldRenderOverline && (
           <div className={overlineClassName}>
-            {label && <Label htmlFor={idVar} text={label} />}
+            {label && (
+              <InputLabel className={labelClassName} htmlFor={idVar}>
+                {label}
+              </InputLabel>
+            )}
             {required && (
-              <Text as="p" size="sm">
+              <Text as="p" className={requiredTextClassName} size="sm">
```

Comment:
it's interesting that we choose the `P` tag here instead of a relatively unstyled `SPAN`. do you remember anything about this choice?

Reply:
Uhhh nothing that I recall 😅